modules/1_bastion/bastion.tf

```diff
@@ -118,6 +118,13 @@ resource "null_resource" "bastion_init" {
         ]
     }
 
+    # We don't need that many threads on the bastion node
+    provisioner "remote-exec" {
+        inline = [
+            "sudo ppc64_cpu --smt=2"
+        ]
+    }
+
```

Comment:
I would prefer not to change SMT settings without some backing data.

Reply:
I ran into timing problems with bootstrapping the master and worker nodes a couple of times as some bootstrap artefacts like grub and ignition files were not available yet. The bootscreen ended in the grub-prompt and manual intervention was needed.
Speeding up using SMT 2 helped a lot.

Reply:
Interesting. Is it possible for you try out SMT=4 in your environment ? In parallel I'll do some runs with SMT=2 as well.
Thanks for this pointer.

Reply:
May be later we can use this as a variable with default to a working level?

Reply:
No timing issues with SMT 2 so far. 
Suggestion: revert to SMT 8 after the upgrade section in modules/5_install/install.tf.